### PR TITLE
feat(academy): enable scheduled quiz report upload

### PR DESCRIPTION
**Pull request by Genyus**
Implements cron job and accompanying WP-CLI command to extract latest quiz activity from Learndash and export to Airtable

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -24,6 +24,10 @@ WP_DEBUG_DISPLAY=false
 # Prevents log being filled with deprecation notices raised by BuddyBoss
 EXCLUDED_ERROR_LEVELS="E_DEPRECATED,E_USER_DEPRECATED"
 
+# Airtable API settings
+AIRTABLE_API_KEY=
+AIRTABLE_REPORTS_BASE=appxMRW8ERvV5bZuw
```
Comment (tanyapowell) [inline .env.example:29]: Is this an example value?

File: web/app/plugins/cbf-multisite/includes/Customizations/Airtable.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+		// convert into structure expected by API
+		$new_activities = array_map(
+			fn ( $result ) => array( 'fields' => $result ),
+			$new_activities
+		);
+
+		// Batch insert activities to reduce number of API calls
+		for ( $i = 0; $i < count( $new_activities ); $i += AIRTABLE_BATCH_SIZE ) {
+			$sub_array = array_slice( $new_activities, $i, AIRTABLE_BATCH_SIZE );
```
Comment (tanyapowell) [inline web/app/plugins/cbf-multisite/includes/Customizations/Airtable.php:92]: What would happen if someone accidentally changes the column order in Airtable?

**Comment by tanyapowell** [comment]
Are there any types of monitoring/alerts that come default with WP-Cron so we can be alerted if a job fails?

File: .env.example (hunk 1 above)
Comment (Genyus) [reply .env.example:29]: No, it's the actual identifier. Do you not want it included?

**Comment by Genyus** [comment]
Depending on what you have in mind, we could use a service like [healthchecks.io](https://healthchecks.io)?

**Comment by tanyapowell** [comment]
Don't worry! Was wondering if there's anything by default or in-built that could be quickly used

File: .env.example (hunk 1 above)
Comment (tanyapowell) [reply .env.example:29]: Yeah, it shouldn't be included in the example file

**Comment by Genyus** [comment]
Yeah, it's just the default WP error logging

File: web/app/plugins/cbf-multisite/includes/Customizations/Airtable.php (hunk 2 above)
Comment (Genyus) [reply web/app/plugins/cbf-multisite/includes/Customizations/Airtable.php:92]: Column reordering is fine, but renaming any exported fields will require code changes

**Review by tanyapowell — APPROVED** [review]

**Pull request merged**